### ISSUE TITLE
NameError in bootstrap script

### ISSUE THREAD

**Issue by tjstum** [pyinstaller/pyinstaller#1918]
This has been mentioned in a couple of other issues (#1891, #1871, #1863), but I don't think it's gotten its own issue.

7933cbbe1fabb2150c74be95be49a3049b304aa4 (related to #1847) appears to break the bootstrap process at least on Windows. A trivial PyInstaller program named `test.py`:
```python
def main():
    print "success!"

if __name__ == "__main__":
    main()
```
Built as `pyinstaller --clean -y test.py`.
Runs fine from PyInstaller installed at dc88bfd5423e3. Breaks on 7933cbbe1fabb:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'c' is not defined
pyiboot01_bootstrap returned -1
```
I do not speak the CPython API all that well, so it's not super clear to me where the exact problem is. I'm happy to try various things if people have ideas but not a great way to (e.g.) test on Windows.

Thanks in advance!

**Comment by htgoebel** [comment]
When using the current development version on Windows, you need to compiler the bootloader.

**Comment by matysek** [comment]
@tjstum Thaks for reporting. I just pushed some changes that also recompile windows bootloader.

This should be fixed in 6c0c40fd3cbecb6b619bf3af1dca0998b591a81c
